Naming.md

```diff
@@ -6,7 +6,15 @@ In general, follow the [official guidelines](https://swift.org/documentation/api
 
 ### Naming a file
 
-There's no three-letter prefix in Swift file naming. For example, if an Objective-C file named XYZEvent.m is to be converted to Swift, it should be renamed to Event.swift.
+- All Swift files should use the `.swift` extension.
+- The name of a file should make it clear what the contents are. For example, if there is a single class named `EasyTapButton` then the file should be called `EasyTapButton.swift`. If you have a view controller, along with some helper types, then using the name of the view controller will likely be the best choice. A good heuristic to use is that if it's ever unclear what to name a file, it may be worth splitting that file up.
+- There's no three-letter prefix in Swift file naming. For example, if an Objective-C file named XYZEvent.m is to be converted to Swift, it should be renamed to Event.swift.
+- If a file contains only one or more extensions, then the file name should be `MyType+Extensions.swift` unless the extension have a specific name; in that case the extension should be named `MyType+ExtensionName.swift`.
+
+### Naming a class, a struct or a protocol
```

Comment:
Should we put a very basic example here?

Reply:
An example for "Naming a class, a struct or a protocol"?

Reply:
I was fooled by some collapsed lines in the PR diff view. Looks good!